### PR TITLE
Add MCP4725

### PR DESCRIPTION
**Pull request by 0xdec**
Add MCP4725, a 12-bit I2C DAC ([datasheet](http://ww1.microchip.com/downloads/en/DeviceDoc/22039d.pdf))

MCP4725

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the symbol(s) you are contributing
- [x] An example screenshot image is very helpful
- [x] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [x] If there are matching footprint PRs, provide link(s) as appropriate
- [x] Check the output of the Travis automated check scripts - fix any errors as required
- [x] Give a reason behind any intentional library convention rule violation.

### CODE REVIEW

**Comment by aewallin** [comment]
S6.3.1.a "For symbols with a default footprint, the simplified footprint name should be appended to the description"

so you should end the description with e.g. ", SOT-23-6"

**Comment by 0xdec** [comment]
@aewallin fixed

File: Analog_DAC.dcm [hunk 1]
```diff
@@ -522,6 +522,12 @@ K 8 bit multiplying DAC
 F http://www.pinballpcb.com/datasheets/OnMC1408-dac.pdf
 $ENDCMP
 #
+$CMP MCP4725
+D 12-bit Digital-to-Analog Converter, external reference, integrated EEPROM, I2C interface, SOT-23-6
+K dac i2c
```
Comment (chschlue) [inline Analog_DAC.dcm]: Remove `i2c`, add `twi`

File: Analog_DAC.dcm [hunk 2]
```diff
@@ -522,6 +522,12 @@ K 8 bit multiplying DAC
 F http://www.pinballpcb.com/datasheets/OnMC1408-dac.pdf
 $ENDCMP
 #
+$CMP MCP4725
```
Comment (chschlue) [inline Analog_DAC.dcm]: Rename to `MCP4725xxx-xCH`

File: Analog_DAC.dcm [hunk 3]
```diff
@@ -522,6 +522,12 @@ K 8 bit multiplying DAC
...
+D 12-bit Digital-to-Analog Converter, external reference, integrated EEPROM, I2C interface, SOT-23-6
```
Comment (chschlue) [inline Analog_DAC.dcm]: I find _external reference_ confusing, Vdd just doubles as Vref.

**Comment by 0xdec** [comment]
@chschlue made requested changes

**Comment by chschlue** [comment]
Thanks.

**Pull request merged**